CODEOWNERS

```diff
@@ -223,15 +220,14 @@
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
-* @cilium/tophat
-/AUTHORS @cilium/tophat
+/AUTHORS @cilium/release-managers
```

Comment:
i'm updating the team configuration for release-managers team. stay tuned.

https://github.com/orgs/cilium/teams/release-managers